autogluon/utils/tabular/ml/learner/default_learner.py

```diff
@@ -28,9 +28,10 @@ def __init__(self, trainer_type=AutoTrainer, **kwargs):
         self.trainer_type = trainer_type
 
     # TODO: Add trainer_kwargs to simplify parameter count and extensibility
-    def _fit(self, X: DataFrame, X_val: DataFrame = None, scheduler_options=None, hyperparameter_tune=False,
+
+    def fit(self, X: DataFrame, X_val: DataFrame = None, X_unlabeled: DataFrame = None,scheduler_options=None, hyperparameter_tune=False,
             feature_prune=False, holdout_frac=0.1, num_bagging_folds=0, num_bagging_sets=1, stack_ensemble_levels=0,
-            hyperparameters=None, ag_args_fit=None, excluded_model_types=None, time_limit=None, save_data=False, save_bagged_folds=True, verbosity=2):
+            hyperparameters=None, ag_args_fit=None, excluded_model_types=None, time_limit=None, save_data=False, save_bagged_folds=True, verbosity=2, unlabeled_data=None):
```

Comment:
Indeed. I added in this extra variable name "unlabeled_data" only because to me, it was a "nicer" name for the input argument to task.fit than "X_unlabeled". I changed the name thereafter to "X_unlabeled" so as to be in line to the naming conventions in the repository. 

In short, it was a stylistic choice; so if you would prefer something different I am quite happy to modify it.

Reply:
sticking with X_unlabeled seems more in line with the other args to me, so I'd remove `unlabeled_data`

Reply:
Removed extraneous parameter.